frontend/projects/upgrade/src/app/features/dashboard/home/components/modal/import-experiment/import-experiment.component.ts

```diff
@@ -485,6 +511,7 @@ export class ImportExperimentComponent implements OnInit {
     result = this.deduceConditionPayload(result);
     result = this.deducePartition(result);
     result = this.deduceFactors(result);
+    result = this.deduceParticipants(result);
```

Comment:
result itself is getting mutated inside the deduce*** functions. We don't need to return anything from the function.

Reply:
code updated